lib/sql/escape.go

```diff
@@ -24,8 +24,10 @@ func NeedsEscaping(name string, destKind constants.DestinationKind) bool {
 	var reservedKeywords []string
 	if destKind == constants.Redshift {
 		reservedKeywords = constants.RedshiftReservedKeywords
-	} else if destKind == constants.MSSQL {
-		return !strings.HasPrefix(name, constants.ArtiePrefix)
+	} else if destKind == constants.MSSQL || destKind == constants.BigQuery {
+		if !strings.HasPrefix(name, constants.ArtiePrefix) {
```

Comment:
In the case that the name does start with `constants.ArtiePrefix`, I changed this so that don't immediately return so that we still run the logic for checking `symbolsToEscape`.

Reply:
Why do that? There should never be reserved keywords that have the Artie prefix

Reply:
So that `__artie_foo:bar` still gets escaped.